server.js

```diff
@@ -0,0 +1,142 @@
+import express from "express";
+import bodyParser from "body-parser";
+import axios from "axios";
+import cors from "cors";
+import dotenv from "dotenv";
+dotenv.config();
+const app = express();
+const port = 3000;
+const API_BASE_URL = "https://io-test.uitdatabank.be";
+const USER_ACCESS_TOKEN = process.env.USER_ACCESS_TOKEN;
+const API_KEY = process.env.API_KEY;
+const EVENT_ID = process.env.EVENT_ID;
+
+const fetchData = async () => {
+  try {
+    const response = await axios.get(
+      `${API_BASE_URL}/labels/?limit=6&query=hello&start=0&suggestion=true`,
+      {
+        headers: {
+          Authorization: `Bearer ${USER_ACCESS_TOKEN}`,
+          "X-Api-Key": API_KEY,
+        },
+      }
+    );
+
+    return response.data;
+  } catch (error) {
+    console.error("Error fetching labels:", error);
+    throw error;
+  }
+};
+
+app.use(bodyParser.json());
+
+app.use(cors());
+
+app.get("/api/labels", async (req, res) => {
+  try {
+    const labels = await fetchData(req.query.query);
```

Comment:
The query is passed to the fetchData function but the function doesn't accept any arguments, see previous comment about making the query dynamic.